gherkin/ruby/lib/gherkin/errors.rb

```diff
@@ -1,5 +1,7 @@
 module Gherkin
   class ParserError < StandardError; end
+  class AstNodeNotLocatedException < Exception; end
```

Comment:
traditionally errors in ruby all end in error (I'm aware that there are existing ones here that don't), so it's not a biggie if you don't fix

Reply:
Custom exceptions should extend from `StandardError`. [Never extend](https://thoughtbot.com/blog/rescue-standarderror-not-exception) `Exception`.

Reply:
inheritance is done.
For the `Error` vs `Exception`, all (except `ParserError` are called 'XXXException`, so I went with the rest

Reply:
Makes sense for now yeh, top work though, this stuff should get gherkin 9 ready soon :)